client/blocks/daily-post-button/test/helper.js

```diff
@@ -1,6 +1,9 @@
 /**
  * External Dependencies
+ *
+ * @format
```

Comment:
Whoops :)

Reply:
Ugh.  This one isn't fun to fix.  Looking into if i can have `sort-imports` so the trick: https://github.com/Automattic/wp-calypso/pull/18486